site/en/docs/privacy-sandbox/shared-storage/index.md

```diff
@@ -92,7 +92,7 @@ storage worklet.
 
 The shared storage data can be used for:
 
-*  [**URL selection**](/docs/privacy-sandbox/shared-storage/url-selection): 
+*  **URL selection**: 
```

Comment:
Should this be titled frequency control and directed there?

Reply:
This section is talking about the gate rather than the use case, so I think it's okay to leave it as "URL selection".